### PR TITLE
Feature/ados29 terms of use

### PR DESCRIPTION
**Pull request by Arochawms10**

### CODE REVIEW

File: src/app/screens/TermsOfUse/index.js [hunk 1]
```diff
@@ -1,11 +1,82 @@
...
+  );
+};
+const TermsOfUseScreen = () => (
+  <LinearGradientContainer>
+    <Container>
+      <ContainerImage>
+        <ImageLogo resizeMode="contain" source={brandImage} />
+      </ContainerImage>
+      <ContainerTextError>
+        <TextWhite>Terms of use</TextWhite>
```
Comment (usaqui-albert) [inline src/app/screens/TermsOfUse/index.js:35]: I think is Title case, "Terms of Use" instead of "Terms of use"

**Comment by usaqui-albert** [comment]
The name of the file should be `terms-icon@2x.png`, you are missing the "x". @Arochawms10

File: src/app/screens/TermsOfUse/index.js [hunk 2]
```diff
@@ -1,11 +1,82 @@
+// @flow
 import React from 'react';
-import { Text } from 'react-native';
+import LinearGradientContainer from '@ui/LinearGradientContainer';
+import brandImage from '@images/terms-icon.png';
+import { ScrollView, ToastAndroid } from 'react-native';
+import { ButtomButtomFixed } from '../../components/hocs/bottomButtomFixed';
```
Comment (usaqui-albert) [inline src/app/screens/TermsOfUse/index.js:6]: Why are we still not using `@hocs`?

File: src/app/screens/TermsOfUse/styles.js [hunk 3]
```diff
@@ -2,6 +2,89 @@ import styled from 'styled-components';
...
+  width: 100%;
+  margin: 0 auto;
+  margin-bottom: 20;
+`;
+
+export const IconSize = styled.Image`
+  width: 25;
+  height: 25;
+  margin-vertical: 5;
+  z-index: 10;
```
Comment (jhonwms) [inline src/app/screens/TermsOfUse/styles.js:85]: why?

**Pull request merged**